packages/perseus/src/components/input-with-examples.jsx

```diff
@@ -14,42 +13,59 @@ import MathOutput from "./math-output.jsx";
 import TextInput from "./text-input.jsx";
 import Tooltip from "./tooltip.jsx";
 
+import type {LinterContextProps} from "../types.js";
+
 const {captureScratchpadTouchStart} = Util;
 const MATH = "math";
 const TEXT = "text";
 const TEX = "tex";
 
-class InputWithExamples extends React.Component<$FlowFixMe, $FlowFixMe> {
-    static propTypes = {
-        type: PropTypes.oneOf([MATH, TEXT, TEX]),
-        value: PropTypes.string,
-        onChange: PropTypes.func.isRequired,
-        className: PropTypes.string,
-        examples: PropTypes.arrayOf(PropTypes.string).isRequired,
-        shouldShowExamples: PropTypes.bool,
-        convertDotToTimes: PropTypes.bool,
-        buttonSet: PropTypes.string,
-        buttonsVisible: PropTypes.oneOf(["always", "never", "focused"]),
-        labelText: PropTypes.string,
-        onFocus: PropTypes.func,
-        onBlur: PropTypes.func,
-        disabled: PropTypes.bool,
-
-        // A unique string identifying this InputWithExamples
-        id: PropTypes.string.isRequired,
-        linterContext: PerseusLinter.linterContextProps,
-    };
-
-    static defaultProps: $FlowFixMe = {
+type Props = {|
+    type: "math" | "text" | "tex",
```

Comment:
**thought**: Yeah. With Flow, those `const`s above don't really help anything... in fact, we often remove them and just use the literal value throughout the code when we've typed a prop with a Flow union. 

No action required.